torch/quantization/quantize_script.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+from .QConfig import QConfig
+
+def prepare_script(model, qconfig_dict, inplace=False):
```

Comment:
is there a way to check that argument is the script module?

Reply:
I'll use type to check it